oauth2_provider/checks.py

```diff
@@ -0,0 +1,29 @@
+from django.apps import apps
+from django.core import checks
+from django.db import router
+
+from .settings import oauth2_settings
+
+
+@checks.register(checks.Tags.database)
+def validate_token_configuration(app_configs, **kwargs):
+    breakpoint()
```

Comment:
It should be removed

Reply:
yep, code is not 100% ready for merge. Thanks for checking this.